static/lang/re-en.json

```diff
@@ -1912,6 +1912,10 @@
                     "Poison": "Orange (Poison)",
                     "Cold": "Brown (Cold)"
                 },
+                "ChaplainsCudgel": {
+                    "BowToAHigherPowerLabel": "Chaplain's Cudgel - Bow to a Higher Power",
+                    "CastTwoActionSpellLabel": "Chaplain's Cudgel - Cast Divine Spell 2+ Actions"
```

Comment:
```suggestion
                    "BowToAHigherPowerLabel": "Chaplain's Cudgel — Bow to a Higher Power",
                    "CastTwoActionSpellLabel": "Cast a divine spell that takes two or more actions"
```
The case should be normal sentence case, and I don't think we need to have the item name in the label (Maybe the higher power one, but not the other). The last one is maybe generic enough to go somewhere else, but I don't know how many things would really interact with it anyways so probably fine staying here.